Assets/Content/Items/Consumables/Food/Consumable.cs

```diff
@@ -9,13 +10,16 @@
 using SS3D.Engine.Inventory.Extensions;
 using SS3D.Engine.Substances;
 using UnityEngine;
+using Random = UnityEngine.Random;
 
 
 /// <summary>
 /// This class handles items that can be consumed
 /// they have an substance container which has have it's contents processed when ingested
 /// </summary>
-public class Consumable : Item, IConsumable
+[RequireComponent(typeof(SubstanceContainer))]
+[RequireComponent(typeof(Item))]
+public class Consumable : InteractionSourceNetworkBehaviour, IInteractionTarget, IConsumable
```

Comment:
You're meant to inherit from IInteractionSourceExtension, not InteractionSourceNetworkBehaviour.
Also replace IInteractionTarget with InteractionTargetNetworkBehaviour.